Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -200,6 +200,10 @@ export default function ModelExperimental(options) {
   this._silhouetteSize = defaultValue(options.silhouetteSize, 0.0);
   this._silhouetteDirty = false;
 
+  // If silhouettes are used for the model, this set to a number coresponding to
+  // the stencil buffer used for rendering the silhouette.
```

Comment:
I see that this value is set outside the class. Maybe make a mention of that here?